lab-dustin/src/component/note-list-container/index.js

```diff
@@ -0,0 +1,25 @@
+import './_note-list-container.scss';
+import React from 'react';
+import uuid from 'uuid/v1';
+import NoteCreateForm from '../note-create-form-container';
+import NoteItem from '../note-container';
+
+class NoteListContainer extends React.Component {
+  constructor(props){
+    super(props);
+    console.log('app', this.props);
+  }
+  render() {
+    let notes = this.props.notes || [];
+    console.log('NOTESSSSSSSSSSSSSSS', this.props);
+    return (
+      <ul>
+        {notes.map(note => {
+          return <NoteItem key={note.id} note = {note} />;
```

Comment:
This line is causing it to just recreate the NoteContainer. Thats why it is just making duplicates of the same thing. If you just replace this with <li>'s with the note content it should start making new list items correctly.